### PR TITLE
[Not tested] K8s acceptance tests and using VSM config to start OpenShift

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
… start OpenShift

Signed-off-by: Lalatendu Mohanty lmohanty@redhat.com

### CODE REVIEW

**Comment by LalatenduMohanty** [comment]
I have not tested the PR yet. So please do not merge.

File: features/openshift.feature [hunk 1]
```diff
@@ -18,11 +18,7 @@ Feature: Command output from various OpenShift related commands
       config.vm.box_url = 'file://../boxes/<box>-<provider>.box'
       config.vm.network :private_network, ip: '<ip>'
       config.vm.synced_folder '.', '/vagrant', disabled: true
-      config.servicemanager.services = 'docker'
-      config.vm.provision "shell", inline: <<-SHELL
-        systemctl enable openshift 2>&1
-        systemctl start openshift | true
-      SHELL
+      config.servicemanager.services = 'docker, openshift'
```
Comment (brgnepal) [inline features/openshift.feature:21]: I guess `openshift` would be enough as docker will run by default. could you check and update?

File: features/service-operation.feature [hunk 2]
```diff
@@ -125,6 +125,19 @@ Feature: Command output from service operations like stop/start/restart
     Then the service "docker" should be running
     And have a new pid for "docker" service
 
+    When the "kubernetes" service is running
```
Comment (brgnepal) [inline features/service-operation.feature:128]: At this moment, kubernetes is running?

**Comment by hferentschik** [comment]
add to whitelist

File: features/service-operation.feature (hunk 2 above)
Comment (hferentschik) [inline features/service-operation.feature:128]: There is are notes in the underlying step definitions (env.rb) that these steps are not working for Kubernetes. If they do the note should be removed

Comment (hferentschik) [inline features/service-operation.feature:128]: I don't think this is going to work as it stands unless Kubernetes is managed by _systemctl_.

**Comment by hferentschik** [comment]
@LalatenduMohanty what's the Github issue for this pull request?

**Comment by hferentschik** [comment]
In its current form not mergable. Closing for now.

**Comment by LalatenduMohanty** [comment]
@hferentschik Interestingly I did not realize till now that this PR is also closed. I was planning to address the review comments now then saw the PR is closed.  Do we really give two days for people to respond. I choose not respond to the review comments in past two days as I had other high priority issues to look in to and I did not want to context switch. 
Now to send a updated change set I need to create another branch and then send  a new PR instead of working on this. To me it gives a message that my PR was not needed / not a good enough feature which is desired. So I am not sure if I want to resend the PR or do something else with the  time I have.

**Pull request closed without merging**